### PR TITLE
Update v2ray and hysteria versions

### PR DESCRIPTION
**Pull request by stevenmcdonald**
update v2ray version for security, ~update hysteria along the way~

I guess I failed at updating Hysteria for the build, and it fails with 1.1.0... so let's just do the security fixes for v2ray for now, one of us can look at updating Hysteria to the latest soon

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -64,11 +65,11 @@ else
     # No .git directory - That's a normal install.
     git clone https://github.com/HyNetwork/hysteria.git "$TMPDIR/hysteria"
     cd hysteria || exit 1
-    git checkout --force --quiet da16c88
+    git checkout --force --quiet 160145f7
     cd ..
     git clone https://github.com/v2fly/v2ray-core.git "$TMPDIR/v2ray-core"
     cd v2ray-core || exit 1
-    git checkout --force --quiet b4069f74
+    git checkout --force --quiet 9b526285
```
Comment (mnbogner) [inline build.sh:72]: i see the hysteria submodule update below but no v2ray submodule update

File: IEnvoyProxy/go.mod [hunk 2]
```diff
@@ -7,14 +7,14 @@ replace (
 	github.com/lucas-clemente/quic-go => github.com/tobyxdd/quic-go v0.27.1-0.20220516000630-9265b64059b0
 	github.com/pion/dtls/v2 => github.com/pion/dtls/v2 v2.0.12
 	github.com/tobyxdd/hysteria v1.0.5 => ../hysteria
```
Comment (mnbogner) [inline IEnvoyProxy/go.mod:9]: does this reference and the one below need to be updated to v1.1.0?

File: build.sh (hunk 1 above)
Comment (stevenmcdonald) [reply build.sh:72]: I see both :) (I did forget to push up the v2ray-core change at first, so maybe you got in before that)

File: IEnvoyProxy/go.mod (hunk 2 above)
Comment (stevenmcdonald) [reply IEnvoyProxy/go.mod:9]: probably... though it built and there's only one version in `../hysteria` ¯\_(ツ)_/¯ I'll take a look at it

Comment (stevenmcdonald) [reply IEnvoyProxy/go.mod:9]: Huh, I hadn't changed the import either, or it got reverted. Rushing too much, probably :)

**Comment by stevenmcdonald** [comment]
I reverted the Hysteria change, so I think this is ready to go

File: build.sh (hunk 1 above)
Comment (mnbogner) [reply build.sh:72]: now i see v2ray but no hysteria, but you said you were reverting hysteria so i guess that makes sense?

**Comment by mnbogner** [comment]
yeah, looks ok and rc worked when i tested it. :+1:

**Review by mnbogner — APPROVED** [review]

**Pull request merged**